### PR TITLE
Portfolio: desktop version

### PR DESCRIPTION
**Pull request by vabuyia**
-black border around input fields has been removed.
-Breaking form has been rectified.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,394 +2,275 @@
...
+          If you have an application you are interested in developing, a feature
+          that you need built or a work-section-li that needs coding. I’d love to help
+          with it
+        </p>
+
+        <form
+          action=" https://formspree.io/f/xqkwojbj"
+          class="form-contact"
+          method="post"
+        >
```
Comment (omar-labana) [inline index.html:234]: - [ ] _[OPTIONAL]_ The `Contact me` section is missing some background images, adding them will make your portfolio look much more appealing.

- [ ] On both screens, your `contact me` form is a bit different than the Figma design(some input fields are not styled, and the `Get in touch` button should be centered), kindly make sure that your form matches the Fifma design as much as you can.
Your design
image
Figma design
image

**Review by omar-labana — CHANGES_REQUESTED** [review]
Hi @vabuyia , 

Excellent work so far! 👌
You have implemented almost all the required features. ✔️ 
There are some cases that you need to adjust but you are almost done! 😀

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -2,394 +2,275 @@
...
+            <li class="box1"><a href="https://twitter.com/abuyiag" target="_blank" rel="noopener noreferrer"><img src="img/twitter.png" alt=" twitter" ></a></li>
+
+            <li class="box2"> <a href="https://linkedin.com/in/vincent-abuya-a1940555" target="_blank" rel="noopener noreferrer"><img src="img/linkedln.png" alt="Linkedln" ></a></li>
+
+            <li class="box3"><a href="https://facebook.com" target="_blank" rel="noopener noreferrer"><img src="img/m.png" alt="m-chat" ></a></li>
+
+            <li class="box4"><a href="https://snapchat.com" target="_blank" rel="noopener noreferrer"> <img src="img/cat.png" alt=" ychat"> </a> </li>
+
+            <li class="box5"><a href="instagram.com" target="_blank" rel="noopener noreferrer"><img src="img/ychat.png" alt=" ycat"></a></li>
+        </ul>
```
[the remark this answers is not in the document]
Comment (vabuyia) [reply index.html:45]: Spacing on the apps has been included in the app's class.

File: index.html [hunk 3]
```diff
@@ -2,394 +2,275 @@
...
```
[the remark this answers is not in the document]
Comment (vabuyia) [reply index.html:180]: Spacing on the apps has been included in the app's class and is properly aligned.

File: index.html [hunk 4]
```diff
@@ -2,394 +2,275 @@
...
+          </div>
+          <hr />
+        </li>
+        <li class="skills">
+          <div class="user-tool-d">
+            <h3>Skills</h3>
+            <i class="fas fa-chevron-right"></i>
+          </div>
+          <hr />
+        </li>
```
[the remark this answers is not in the document]
Comment (vabuyia) [reply index.html:217]: Arrows on both Languages, Framework, and skills sections have been included.

File: index.html (hunk 1 above)
Comment (vabuyia) [reply index.html:234]: Proper styling has been added to the form

File: index.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (vabuyia) [reply index.html:45]: -A spacing to social-media apps.

File: index.html (hunk 4 above)
[the remark this answers is not in the document]
Comment (vabuyia) [reply index.html]: -Add arrows to the Languages, Frameworks, and skills section.

File: index.html (hunk 1 above)
Comment (vabuyia) [reply index.html:234]: -Proper styling of the form section.

File: index.html [hunk 5]
```diff
@@ -2,394 +2,275 @@
...
+              <textarea id= "form-textarea"
+                name="message"
+                placeholder="Write your message here"
+                id="message"
+                cols="30"
+                rows="10"
+                maxlength="500"
+                required
+              ></textarea>
             </li>
```
Comment (omar-labana) [inline index.html:266]: - [ ] On the mobile screen, input fields are breaking your design causing horizontal scrolling, also there is a black border around the input fields, kindly investigate the issue making sure it matches the Figma design as much as possible.
image

**Review by omar-labana — CHANGES_REQUESTED** [review]
Hi @vabuyia, 

Great work on making the changes requested previously👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_N/A_
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (vabuyia) [reply index.html:234]: -black border around input fields has been removed.
-Breaking form has been rectified.

**Review by omar-labana — APPROVED** [review]
Hi @vabuyia, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**